@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/damage/fn_damageApply.sqf

```diff
@@ -56,6 +56,16 @@ if (isEngineOn _heli) then {
                             \nDMG = %3", _pctNR, _engPctTQ, _dmg];
             */
         };
+
+        if (getpos _heli select 2 > 1) then {
+            private _dmg = _totRtrDmg + (_dmgPerSec * _deltaTime);
```

Comment:
Rather having two systems with the same contents, would it be possible to `||` together the two conditions?